### DISCUSSION TITLE
does carousel works well after update?

### DISCUSSION THREAD

**Discussion by Xyzjesus** [mantinedev/mantine#4796] [Feedback]
hi friends! does carousel v7.0 works well?

**Comment by rtivital** [comment]
It works fine on the docs website

**Reply by Xyzjesus** [reply]
hi! thanks for answer. i'm curious because after update from 6 to 7 it returns me only vertical with all props i'm attaching 
<img width="1419" alt="Screenshot 2023-09-20 at 09 41 07" src="https://github.com/mantinedev/mantine/assets/62913761/6100a75e-21ab-4ea4-824e-f7bd09d5daf9">
<img width="530" alt="Screenshot 2023-09-20 at 09 42 58" src="https://github.com/mantinedev/mantine/assets/62913761/4addc156-0dbf-4904-9ae6-970d599fca8d">

**Reply by rtivital** [reply]
You did not import `import '@mantine/carousel/styles.css';` like it is shown in documentation

**Reply by Xyzjesus** [reply]
oh yes! thanks a lot
